admin/afs-client-accessd/afs-client-accessd

```diff
@@ -601,6 +611,11 @@ read_config()
 		$newcfg{$_} = $override{$_};
 	}
 
+	if ($newcfg{EXPORT_TIME} < 0 || $newcfg{EXPORT_TIME} > 24) {
+		$newcfg{EXPORT_TIME} = 0;
```

Comment:
Not sure if possible, but it would be better if we could log a warning if changing a value which is out of range.